index.html

```diff
@@ -0,0 +1,206 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet"/>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+  <title>NIGERIA MUSIC SUMMIT</title>
+</head>
+<body>
+  <header>
+    <aside class="top-section">
+      <div class="bar-container">
+      <ul class="bars">
+        <li>
+          <i class="fab fa-facebook-f"></i>
+        </li>
+        <li>
+          <i class="fab fa-twitter"></i>
+        </li>
+        <li>
+          <h4>English</h4>
+        </li>
+        <li>
+          <h4>My Page</h4>
+        </li>
+        <li>
+          <h4>Logout</h4>
+        </li>
+      </ul>
+      </div>
+    </aside>
+    <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html"><img src="./image/logo.jpg" alt="LOGO" class="logo"></a>
+          <div class="menu-bar nav" id="menu">
+            <div class="icon"></div>
+            <div class="icon"></div>
+            <div class="icon"></div>
+          </div>
+          <div class="cancel" id ='close'>
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Securam Circle</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+
+<section class="home-page">
+  <div class="title">
+    <h1 class="intro-headline">"Hello!! It's Afrobeat to the world"</h1>
+    <h1 class="main-headline" >NIGERIA MUSIC SUMMIT 2021</h1>
+    <div class="theme">
+      <p>
+        Celebration of Nigerian hiphop miusic, its global reception, and challenges. creating a positive change in Nigerian music industry.
+      </p>
+      </div>
+      <div class="date">
+       <h2> 2021.12.16(THU) ~ 17(FRI) </h2>
+       <p>@ Eko Hotels & Suites <br> 1415 Adetokunbo Ademola Street, <br>Victoria Island Lagos </p>
+      </div>
+  </div>
+</section>
+
+<!-- About section -->
+
+<section class="program">
```

Comment:
- [ ] The space between these should be smaller. Kindly reduce the space among them in  order to match the given deisgn
![Screenshot from 2021-11-18 19-17-49](https://user-images.githubusercontent.com/62705947/142473698-6503a085-6878-41a2-af37-cd002084713a.png)

Reply:
- [ ] For mobile version please adjust the margins/padding in order to match the given design:

Yours:
![Screenshot from 2021-11-18 19-25-00](https://user-images.githubusercontent.com/62705947/142474750-b80c8a51-c21c-44ed-8dd0-f44e15ea2ec2.png)
Original
![Screenshot from 2021-11-18 19-24-58](https://user-images.githubusercontent.com/62705947/142474789-b8446d2e-5a6a-40ee-8d8c-f5e0f3692c37.png)

Reply:
Well solved